Documentation/PageTsconfig/Mod/WebLayout/BackendLayout.rst

```diff
@@ -0,0 +1,102 @@
+..  include:: /Includes.rst.txt
+
+..  index::
+    mod; web_layout
+    Modules; Page
+..  _backend-layouts:
```

Comment:
Can we add a little bit of space here?

Reply:
I removed the index as it was only copy-pasted and has no real content. However I cannot see where space is or was missing?

Reply:
After removing the index it looks good again.